app/controllers/api/base_controller.rb

```diff
@@ -9,6 +11,10 @@ class Api::BaseController < ApplicationController
 
 private
 
+  def set_cache_headers
+    expires_in 24.hours, public: true
```

Comment:
The data is updated daily, but It might be more accurate if we expire it at the end of the day, as we will have fresh data every day at 1am.

Reply:
How do you want this to work?

Do you mean something like this?

```
    hour = Time.now.utc.hour
    expiry_hour = 1
    cache_duration = (expiry_hour - hour) % 24
    expires_in cache_duration.hours, public: true
```

Reply:
yes, something like:

```ruby
expiry_time = Time.zone.tomorrow.at_beginning_of_day.change(hour: 1)
current_time = Time.zone.now 

hours = ((expiry_time - current_time) / 1.hour).round
```